### PR TITLE
Development

### PR DESCRIPTION
**Pull request by madcido**

### CODE REVIEW

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello guys!! 

I can't see a link to the site you're cloning in your README.md file. You need to clone a real site good enough to show to a potential employer. In my opinion, it will be better if you choose another site to clone as this project does not reflect most of the concepts you have learnt in this section.

However, please add the link to the site you're cloning in the README.md file.

When you're done, request for another code review and add me as TSE so I can follow up on it. 

Happy coding:sunglasses:

**Comment by madcido** [comment]
@JamesOkunlade 
Thanks for your review. There is no link to the original site because this is not a clone. In my opinion, this site does have some of the concepts learnt in this section: round borders, box shadows, pseudo-classes and even a pop-up menu made of pure CSS, besides being built with a grid-based framework. Also, if you want to see some animations, check this old project: https://madcido.github.io/sm-clone/. The Odin Project said to clone their own site if we were out of creativity, and we are, so that's what we are going to do. I hope a clone from The Odin Project will reflect this concepts better. If it doesn't, then maybe Microverse should make their own course and/or be more specific about the assignments.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Superb work guys!!! :v:

Hey guys, its amazing to see what you can achieve with everything you've learnt. You did a good job with the odin project. You should check the responsiveness though. You need to make most divs have block views like the mobile view of the main site.

Also, you should keep you code clean of all other codes from the previous attempt.

Please do this as soon as you can and request for most likely the last code review.

Happy coding guys. :sunglasses:

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+}
+
+.odin-footer .social-link i {
+  margin: 0 15px;
+  font-size: 25px;
+  color: #ce973e;
+}
+
+.odin-footer .social-link i:hover {
+
```
Comment (JamesOkunlade) [inline css/custom.css]: Remove this white space

File: css/custom.css [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  .odin-nav-button{
+    display: block;
+  }
+  .outro1 div{
+    display: block;
+  }
+  .outro3 div{
+    display: block;
+    flex-direction: column;
+
```
Comment (JamesOkunlade) [inline css/custom.css]: Remove this white space

**Review by JamesOkunlade — APPROVED** [review]
### Perfect!!!

Hey guys, congrats on finishing your last project on HTML & CSS section. Please merge to master, close pull request and fill the completion form.

Here are your thumbs :+1: :+1:

Good luck with your future projects.

**Pull request merged**